packages/tradershub/src/constants/constants.tsx

```diff
@@ -106,7 +106,7 @@ export const IconToCurrencyMapper: IconToCurrencyMapperType = {
         icon: <CurrencyUsdcIcon />,
         text: 'USD Coin',
     },
-    USDT: {
+    UST: {
```

Comment:
`Tether` is supposed to be `USDT` right?

Reply:
There is no such currency coming from BE. Its UST